src/convertToObject.js

```diff
@@ -5,8 +5,25 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(str) {
+  const resultStyles = {};
+  const listStyle = str.split(';');
```

Comment:
Splitting the string by semicolons is a good start, but you need to handle potential empty entries and trim whitespace for each style declaration for accurate processing.